aiven/client/cli.py

```diff
@@ -3352,6 +3361,7 @@ def project__create(self):
                 vat_id=self.args.vat_id,
                 billing_emails=self.args.billing_email,
                 tech_emails=self.args.tech_email,
+                use_source_project_billing_group=self.args.use_source_project_billing_group
```

Comment:
tiny nitpick: I'd put terminating comma to make potential next parameter addition one-liner.

Reply:
Fixed